src/apis/Minecraft.js

```diff
@@ -0,0 +1,33 @@
+const { APIWrapper } = require('../')
+const axios = require('axios')
+const moment = require('moment')
+
+module.exports = class Minecraft extends APIWrapper {
+  constructor () {
+    super({
+      name: 'minecraft'
+    })
+  }
+
+  async nameToUUID (name) {
+    const { data } = await axios(`https://api.mojang.com/users/profiles/minecraft/${name}?at=${moment().format('x')}`)
```

Comment:
Will data always be an object and not null?

Reply:
probably can be null, but he can handle catch clause(s) I think

Reply:
It may be null, but there's a check after (`if (data.id)`) to see if there's content

Reply:
If its null and it tries to access data.id, then a TypeError will happen, it should check if data isn't null first.

Reply:
If data is null it's probably because of some unexpected error so it'll be already catched by the try catch